cli/cmd/testdata/inject_gettest_deployment.golden

```diff
@@ -0,0 +1,22 @@
+Error: error converting YAML to JSON: yaml: line 14: did not find expected key
+Usage:
```

Comment:
Further thought: Regardless of whether we print the error message to stdout or stderr (and it really should be stderr), we should not print the "Usage" message on this kind of error.